rootfs/etc/confd/templates/builder

```diff
@@ -64,17 +64,30 @@ TMP_DIR=$(mktemp -d -p $BUILD_DIR)
 
 cd $REPO_DIR
 # use Procfile if provided, otherwise try default process types from ./release
+
+git archive --format=tar.gz ${GIT_SHA} > ${APP_NAME}.tar.gz
```

Comment:
why are we archiving then immediately un-tar'ing the directory elsewhere? Can't we just run `cp -R` instead?